Employee.java

```diff
@@ -0,0 +1,65 @@
+public class Employee implements Comparable<Employee> {
+    private int ID;
+    private String NAME;
+    private String Position;
+    private int Salary;
+
+    public int getID() {

```

Comment:
same, should be `getId()` instead of `getID()`
applies to other methods as well

Reply:
fixed on cac633a